docs/learn/calls.md

```diff
@@ -48,15 +50,20 @@ response = recommend_book("fantasy")
 print(response.content)
 ```
 
-In this example, we're using OpenAI's `gpt-4o-mini` model to generate a book recommendation for a user-specified genre. The call decorator transforms the `recommend_book` function into an LLM API call. The function arguments are automatically injected into the prompt defined in the [`@prompt_template`](./prompts.md#prompt-templates) decorator.
+In this example:
+
+- The [`@prompt_template`](./prompts.md#prompt-templates) decorator turns the function into a prompt template.
+- The `@openai.call` decorator transforms the prompt template into an LLM API call.
+- We're using OpenAI's `gpt-4o-mini` model to make the call.
+- When we call the function with `genre="fantasy"`, the argument is automatically injected into the prompt template and used in the call.
 
 !!! info "Function Body"
 
     In the above example, we've used an ellipsis (`...`) for the function body, which returns `None`. If you'd like, you can always explicitly `return None` to be extra clear. For now, you can safely ignore how we use the function body, which we cover in more detail in the documentation for [dynamic configuration](./dynamic_configuration.md).
```

Comment:
nit: as i mentioned in my recent review of the "Prompts" documentation, we should drop the "Dynamic Configuration" page and just discuss it in each respective section.

this means here we should discuss dynamic call parameters in the section on call parameters, and then we should mention that section here instead of the original dynamic configuration docs.

thoughts?